docs/hub/security-two-factor-authentication.md

```diff
@@ -0,0 +1,95 @@
+    - Authy
+    - Google Authenticator
+    - Microsoft Authenticator
+    - FreeOTP
+2. In the application, add a new entry in one of two ways:
+  - Scan the code displayed by Hugging Face Hub with your device’s camera to add the entry automatically
+  - Enter the details provided to add the entry manually
+
+In Hugging Face Hub:
+1. Enter the six-digit pin number from the entry on your device into Pin code.
+2. Save
+
+<!-- screenshot -->
+
+If you entered the correct pin, the Hub displays a list of recovery codes. Download them and keep them in a safe place.
+
+## Recovery codes
+
+Right after you've successfully activated 2FA with a one-time password, you're requested to download a collection of generated recovery codes. If you ever lose access to your one-time password authenticator, you can utilize one of these recovery codes to log in to your account.
+
+- Each code can be used only **once** to sign in to your account
+- You should copy and print the codes, or use "Download codes" to download them for storage in a safe place. If you choose to download them, the file is called **huggingface-recovery-codes.txt**
+
+<!-- blurred screenshot -->
+
+If you lose the recovery codes, or want to generate new ones, you can use the [Authentication settings](https://hf.co/settings/authentication) page.
+
+## Regenerate two-factor authentication recovery codes
+
+To regenerate 2FA recovery codes:
+1. Access your [Authentication settings](https://hf.co/settings/authentication)
+2. If you’ve already configured 2FA, select Recovery Code
+3. Click on Regenerate recovery codes
+
+<Tip warning={true}>
+If you regenerate 2FA recovery codes, save them. You can’t use any previously created recovery codes.
+</Tip>
+
+## Sign in with two-factor authentication enabled
+
+When you sign in with 2FA enabled, the process is only slightly different than the standard sign-in procedure. After entering your username and password, you'll encounter an additional prompt, depending on the type of 2FA you've set up. When prompted, provide the pin from your one-time password authenticator's app or a recovery code to complete the sign-in process.
+
+<!-- screenshot -->
+
+## Disable two-factor authentication
+
+To disable 2FA:
+1. Access your [Authentication settings](https://hf.co/settings/authentication)
+2. Click on "Remove".
+
+This clears all your 2FA registrations.
+
+## Recovery options
+
+If you don’t have access to your code generation device, you can recover access to your account:
+
+- Use a saved recovery code, if you saved them when you enabled two-factor authentication
+- Requesting help with two-factor authentication
+
+### Use a recovery code
+
+To use a recovery code:
+1. Enter your username or email, and password, on the [Hub sign-in page](https://hf.co/login)
+2. When prompted for a two-factor code, click on "Lost access to your two-factor authentication app? Use a recovery code"
+3. Enter one of your recovery code
```

Comment:
```suggestion
3. Enter one of your recovery codes
```

Reply:
If they clicked on "lost access to authentication app", maybe we should tell them to re-create mfa on another app/device? Will they be guided to that process automatically?